DCOS/utils/update-keyvault-secrets.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+if [[ -z $AZURE_KEYVAULT_NAME ]] || [[ -z $PRIVATE_KEY_SECRET_NAME ]] || [[ -z $PUBLIC_KEY_SECRET_NAME ]] || [[ -z $WIN_PASS_SECRET_NAME ]]; then
+    echo "ERROR: KEYVAULT_NAME, PRIVATE_KEY_SECRET_NAME, PUBLIC_KEY_SECRET_NAME and WIN_PASS_SECRET_NAME are mandatory"
+    return 1
+fi
+
+GENERATED_SSH_KEY_PATH="${WORKSPACE}/id_rsa"
```

Comment:
I wouldn't use `$WORKSPACE` variable here. One would want to use this script outside the Jenkins.

I'd suggest you create a temporary directory with `mktemp -d`, use that in the script and clean it up at the end.

Make something like this:
```
TMP_DIR=$(mktemp -d)
GENERATED_SSH_KEY_PATH="${TMP_DIR}/id_rsa"
...
...
... (end of the script) ...
rm -rf $TMP_DIR
```